---
layout: page
permalink: /research/
title: Research
nav: true
nav_order: 1
---



### **The biogeography of _C. difficile_: from CDI to healthy infants**

<img align="left" width="350" height="350" style="float:left; padding-right:15px" src="/assets/img/3.jpg">

_C. difficile_ has been listed by the CDC among the most urgent threats in hospital-acquired infections, and it is often described in the literature as "well known human pathogen" responsible for C. difficile infection (CDI). My PhD work focused on investigating the composition and stability of the gut microbiome in presence of _C. difficile_. To do so, I first studied _C. difficile_ in the most obvious context: CDI patients. However, among those patients, _C. difficile_ carriage was suprisingly low: only 30%. I instead found elevated carriage of other pathogenic species that are known to be able to induce CDI-like symptomatology, suggesting over-diagnosis in the evaluated studies. 
<br/><br/>
I then expanded my scope of study, including not only public metagenomic studies on adult/elderly CDI patients, but also healthy subjects of all ages, animals and the environment. I put together a collection of 42,900 publicly available metagenomic samples, curated their associated metadata (yes was a lot of work!), and then leveraged this dataset to investigate the global biogeography and genetic variability of _C. difficile_. I found that _C. difficile_ is most prevalent (up to 76%) and abundant in healthy infants, and that _C. difficile_ in this age group is associated with multiple indicators of a healthy infant gut microbiome development. **This study represents the largest single-species metagenomic survey to date, providing systematic, world-wide analysis of _C. difficile_ carriage and associated microbiome composition.**
<br/><br/>
_Relevance of the study: Disentangling the real burden of C. difficile from other pathogenic species inducing the same symptomatology will help directing future research focus and fundings towards better diagnostic systems and diagnosis. In addition, understanding how C. difficile carriage remains asymptomatic will help understanding the mechnanisms behind CDI manifestation and progression._


<br/><br/>
### **Mother-to-infant microbiome transmission**

<img align="left" width="400" height="350" style="float:left; padding-right:15px" src="/assets/img/4.jpg"> 

At birth, the infant gut represents a "blank sheet", which the microbes readily colonize. But where are the colonizing microbes coming from? To address this question, I started, first during my Master then in my PhD to investigate the most intimate relationship that the infant has: the one with the mother.  In this study, I investigated how the maternal gut, skin, oral and vaginal microbiomes seed and shape the infant microbiome in the first semester of life. For this study, I collected 2,000+ samples over the course of 3 years, and I worked in close contact with the neonatologists and OB/GYNs at the Santa Chiara Hospital in Trento. This was the project that sparked my interest in microbiome research, and I enjoyed every bit of it.

_Relevance of the study: Understanding the routes of microbial transmission from the mother to her baby helps defining better healthcare policies, and strenghtens the importance of practices part of the [Baby Friendly Hospital Initiative](https://www.unicef.org/media/95191/file/Baby-friendly-hospital-initiative-implementation-guidance-2018.pdf), such as skin-to-skin contact and rooming-in._ 

<br/><br/>
<br/><br/>
### **How to define and identify strains in microbiome research**

<img align="left" width="500" height="350" style="float:left; padding-right:15px" src="/assets/img/7.jpg"> 

Most studies look at species-level composition, but often the presence (or absence) of a single gene, or a set of genes, can make the difference between health and disease. Hence, the importance of diving into the strain-level world. In this work, I explored (with Thea Van Rossum and Oleksandr Maistrenko) the mechanisms that drive variability within bacterial species and the challenges in stratifying such variability, providing harmonized terminology and the associated scope of applicability. This Review, which took more than a year to write and gave us all a fair amount of headaches, touches important aspects of microbial ecology that are widely applicable (from human microbiome, to animals, to marine microbial ecosystems).

_Relevance of the study: Our ability to reproduce results and compare them among studies depends on common terminology (e.g. what is a strain?) and agreement on gold standards (e.g. when are MAGs good enough to be used in strain-level analyses?). Knowing the importance of strain-level analysis, its challenges, and the limitations of the available tools guides the researchers towards better study design, analysis and costs allocation._

<br/><br/>
